src/components/slide-toggle/README.md

```diff
@@ -0,0 +1,66 @@
+# MdSlideToggle
+`MdSlideToggle` is a two-state control, which can be also called `switch`
+
+### Screenshots
+![image](https://raw.githubusercontent.com/angular/code.material.angularjs.org/master/material2_assets/slide-toggle/toggles.png)
```

Comment:
I added a hosted asset for you:
https://material.angularjs.org/material2_assets/slide-toggle/toggles.png